src/uwtools/resources/jsonschema/make-hgrid.jsonschema

```diff
@@ -0,0 +1,119 @@
+{
+  "properties": {
+    "make_hgrid": {
+      "additionalProperties": false,
+      "properties": {
+        "config": {
+          "additionalProperties": false,
+          "properties": {
+            "grid_type": {
+              "type": "string",
+              "enum": [
+                "from_file",
+                "spectral_grid",
+                "regular_lonlat_grid",
+                "tripolar_grid",
+                "conformal_cubic_grid",
+                "gnomonic_ed",
+                "simple_cartesian_grid",
+                "f_plane_grid",
+                "beta_plane_grid"
+              ]
+            },
+            "my_grid_file": { "type": "string" },
+            "nxbnds": { "type": "integer" },
+            "nybnds": { "type": "integer" },
+            "xbnds": { "type": "array", "items": { "type": "number" } },
+            "ybnds": { "type": "array", "items": { "type": "number" } },
+            "nlon": { "type": "array", "items": { "type": "integer" } },
+            "nlat": { "type": "array", "items": { "type": "integer" } },
+            "dlon": { "type": "array", "items": { "type": "number" } },
+            "dlat": { "type": "array", "items": { "type": "number" } },
+            "lat_join": { "type": "number" },
+            "nratio": { "type": "number" },
+            "simple_dx": { "type": "number" },
+            "simple_dy": { "type": "number" },
+            "grid_name": { "type": "string" },
+            "center": { "type": "string", "enum": ["none", "t_cell", "c_cell"] },
+            "shift_fac": { "type": "number" },
+            "do_schmidt": { "type": "boolean" },
+            "do_cube_transform": { "type": "boolean" },
+            "stretch_factor": { "type": "number" },
+            "target_lon": { "type": "number" },
+            "target_lat": { "type": "number" },
+            "nest_grids": { "type": "integer" },
+            "parent_tile": { "type": "array", "items": { "type": "integer" } },
+            "refine_ratio": { "type": "array", "items": { "type": "number" } },
+            "halo": { "type": "integer" },
+            "istart_nest": { "type": "array", "items": { "type": "integer" } },
+            "iend_nest": { "type": "array", "items": { "type": "integer" } },
+            "jstart_nest": { "type": "array", "items": { "type": "integer" } },
+            "jend_nest": { "type": "array", "items": { "type": "integer" } },
+            "great_circle_algorithm": { "type": "boolean" },
+            "out_halo": { "type": "integer" },
+            "angular_midpoint": { "type": "boolean" },
+            "non_length_angle": { "type": "boolean" },
+            "rotate_poly": { "type": "boolean" },
+            "verbose": { "type": "boolean" }
+          },
+          "required": ["grid_type"],
```

Comment:
I'm currently having issues with the formatter removing my conditionals here, I'm working on a solution but have not solved it yet.

Reply:
Formatter fixed by fixing some issues with my conditionals!